Tests/Provider/AbstractProviderTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\Provider;
```

Comment:
`Test` is for abstract test class that can be used publicly. This means by other sonata bundles and other user projects. This part is autoloaded.

`Tests` is for internal test cases of the library. This part is not auto-loaded on production.